sales.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title></title>
```

Comment:
We should add a title here (or remove the element altogether, though I think a value would be nice).